packages/lesswrong/lib/collections/bookmarks/newSchema.ts

```diff
@@ -0,0 +1,103 @@
+import { DEFAULT_CREATED_AT_FIELD, DEFAULT_ID_FIELD } from "@/lib/collections/helpers/sharedFieldConstants";
+import { userOwns } from "@/lib/vulcan-users/permissions";
+import gql from "graphql-tag";
+
+const ALLOWED_COLLECTION_NAMES = ["Posts", "Comments"];
+
+export const graphqlTypeDefs = gql`
+  union BookmarkableDocument = Post | Comment
```

Comment:
This isn't getting used anywhere, as far as I can tell?  (Also I don't think we support union types anywhere else in our code right now and it doesn't seem worth the overhead.)